src/olgm/herald/TileSource.js

```diff
@@ -224,7 +224,7 @@ class TileSourceHerald extends SourceHerald {
       }
     }
 
-    let result = getTileUrlFunction(ol3Coords, 1, proj);
+    let result = source.tileUrlFunction(ol3Coords, 1, proj);
 
     // TileJSON sources don't have their url function right away, try again
     if (result === undefined) {
```

Comment:
Should the method below be changed too?

Reply:
we can keep it for backward compatibility, ol version  6.1.1

Reply:
we can keep it for old version (6.1.1) to function.